decidim-initiatives/spec/controllers/decidim/initiatives/admin/initiatives_types_controller_spec.rb

```diff
@@ -9,8 +9,8 @@ module Admin
         routes { Decidim::Initiatives::AdminEngine.routes }
 
         let(:organization) { create(:organization) }
-        let(:admin_user) { create(:user, :confirmed, :admin, organization:) }
-        let(:user) { create(:user, :confirmed, organization:) }
+        let(:admin_user) { create(:user, :confirmed, :admin, admin_terms_accepted_at: Time.current, organization:) }
+        let(:user) { create(:user, :confirmed, admin_terms_accepted_at: Time.current, organization:) }
```

Comment:
You can also use the `:admin_terms_accepted` trait here.
```suggestion
        let(:user) { create(:user, :confirmed, :admin_terms_accepted, organization:) }
```